doc/user/content/ops/monitoring.md

```diff
@@ -175,3 +175,110 @@ Configuration parameter | Value
 `prometheus_url`        | `http://<materialized host>/metrics`
 `namespace`             | Your choice
 `metrics`               | `[mz*]` to select all metrics, or a list of specific metrics
+
+## Logging
+
+Materialize periodically emits messages to its [log file](/cli/#log-filter).
+These log messages serve several purposes:
+
+  * To alert operators to critical issues
+  * To record system status changes
+  * To provide developers with visibility into the system's execution when
+    troubleshooting issues
+
+As an operator, you should monitor for messages at the [`WARN` or `ERROR`
+levels](#levels). Every message at either of these levels indicates an
+issue that must be investigated and resolved.
+
+### Message format
+
+Each log message is emitted on its own line with the following format:
+
+```
+<timestamp> <level> <module>: [<tag>]... <body>
+```
+
+For example, Materialize emits the following message when a table named `t`
+is created:
+
+```
+2021-04-08T04:12:25.927738Z  INFO coord::catalog: create table materialize.public.t (u1)
+```
+
+The timestamp is always in UTC and formatted according to ISO 8601.
+
+The log level is one of the five levels described in the next section,
+formatted with all uppercase letters.
+
+The module path reflects the log message's location in Materialize's source
+code. Module paths change frequently from release to release and are not part of
+Materialize's stable interface.
+
+The tags, if included, further categorize the message. Tags are surrounded by
+square brackets. The currently used tags are:
+
+* `[customer-data]`: the message includes clear-text contents of data in the system
+* `[deprecation]`: a feature in use will be removed or changed in a future release
+
+The body is unstructured text intended for human consumption. It may include
+embedded newlines.
+
+### Levels
+
+Every log message is associated with a level that indicates the severity of the
+message. The levels are, in decreasing order of severity, `ERROR`, `WARN`,
+`INFO`, `DEBUG`, and `TRACE`.
+
+Log levels are used in the [`--log-filter` command-line option](/cli/#log-filter)
+to determine which log messages to emit.
+
+Messages at each level must meet the indicated standard:
+
+* **`ERROR`**: Reports an error that has caused data loss, corruption, or
+  unavailability. You should page on-call staff immediately about these errors.
+
```

Comment:
This is a nitpick, but in the first sentence "data" applies to "loss" and "corruption" but not "unavailability" (I think). If so, it could be "Reports an error that has caused data loss or corruption or system unavailability."

Same thing for `WARN`.